index.html

```diff
@@ -14,8 +14,9 @@
             <div id="nav-bar">
                 <a class="nav-link" href = "#landing_page_1">Link 1</a>
                 <a class="nav-link" href = "#landing_page_2">Link 2</a>
-                <a class="nav-link" href = "#landing_page_3">Link 3</a>
-                </div>
+                <a class="nav-link" href = "#landing_page_3">Link 3</a> 
```

Comment:
💪 💪 You are almost there, you did it 💪 💪 
The link you used in link number 3  & 4 are the same, so it is better to different link or removes the one of them

Reply:
- [ ] Please, kindly make this comment more clearer; I don't understand what you meant by "**it is better to different link**". I think our explanation suppose to be easy to understand and clearer to the other party. 👍